action.yml

```diff
@@ -4,14 +4,18 @@ inputs:
   github_token:
     description: 'Token with repository permissions, e.g. secrets.GITHUB_TOKEN of the caller workflow'
     required: true
+  github_head_sha:
+    description: 'Hash of HEAD commit'
+    default: None  # TODO: Remove after merge-check.yml have been changed
+    required: false  # TODO: Set to true after merge-check.yml have been changed
 
 runs:
 
   using: "composite"
 
   steps:
     - name: Setup Python
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v4
```

Comment:
Ok, I'll make sure we have this on our radar. I haven't done much with Dependabot so far, so it might be a good opportunity to understand what we are currently covering and extend that.